web/js/map/ui.js

```diff
@@ -98,6 +99,26 @@ export function mapui(models, config, store, ui) {
    */
   const subscribeToStore = function(action) {
     switch (action.type) {
+      case TOGGLE_DISTRACTION_FREE_MODE: {
+        var map = self.selected;
+        const isDistractionFreeModeActive = store.getState().ui.isDistractionFreeModeActive;
+        if (isDistractionFreeModeActive) {
+          // hide map scale and zoom buttons
+          map.removeControl(map.wv.scaleImperial);
+          map.removeControl(map.wv.scaleMetric);
```

Comment:
Could all of these be hidden with styles by adding a `distraction-mode` class to the outermost map div?